lib/autoproj/package_managers/shell_script_manager.rb

```diff
@@ -3,7 +3,8 @@ module PackageManagers
         # Base class for all package managers that simply require the call of a
         # shell script to install packages (e.g. yum, apt, ...)
         class ShellScriptManager < Manager
-            def self.execute(command_line, with_locking, with_root, env: Autobuild.env)
+            def self.execute(command_line, with_locking, with_root,
+                env: Autobuild.env, inherit: Set.new)
```

Comment:
use `ws.env` instead of `Autobuild.env`

Reply:
?

As the default keyword argument?

Reply:
Yes

Reply:
Then you mean `Autoproj.workspace.env`, right?

Reply:
Nope. Manager has a `#ws` attribute. I've been trying to avoid these globals when doing autoproj 2.0

Reply:
> Manager has a #ws attribute

Yes, but `self.execute` is a class method and `#ws` is an instance method.

Reply:
Ah ... got it. Sorry. Autobuild.env was already there actually ... :p I thought you added it.

Autoproj.workspace.env would probably be better, but the really good one would have been to be mandatory. Oh well.